cmd/baseserver/Makefile

```diff
@@ -0,0 +1,15 @@
+all: install linux darwin windows
```

Comment:
`Do we need those CGO_ENABLED flags ?`
Good question, I set them because in the past I've found more problems with `CGO_ENABLED=0` not being explicitly specified, for example the cgo net DNS resolver being used in networking as well as builds on Windows failing as per https://github.com/odeke-em/drive/issues?utf8=%E2%9C%93&q=is%3Aissue%20CGO_ENABLED.

`Also, these all look like the same command. Do we need them to be separate or can we just have one make command?`
Unfortunately am not that proficient with make and I haven't yet found a way to concisely avoid this repetition.